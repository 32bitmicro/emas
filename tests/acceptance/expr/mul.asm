.word 1 * 1
.word 1.0 * 1.0
.word 1 * 0
.word 1.0 * 0.0
.word -1 * 1
.word -1.0 * 1.0
.word -1 * -1
.word -1.0 * -1.0
.word -1 * 0
.word -1.0 * 0.0
.word 2 * 32767
.word 2.0 * 32767.0
