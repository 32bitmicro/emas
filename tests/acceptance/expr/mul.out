@ 0x0000 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0001 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0002 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0003 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0004 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0005 : 0xffff  /  111 111 1 111 111 111  /  -1
@ 0x0006 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0007 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0008 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0009 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x000a : 0xfffe  /  111 111 1 111 111 110  /  65534
@ 0x000b : 0xfffe  /  111 111 1 111 111 110  /  65534
